### PR TITLE
Rewrite Readme.md

### PR DESCRIPTION
**Pull request by MineKing9534**
## Checklist

 - [x] Read [Contribution Guidelines](https://github.com/SlimeCloud/java-SlimeBot#contributing)
 - [x] Tested the Code
 - [x] This PR is ready to review and merge

## Description
Rewrite and improve the Readme.md file to be more precise

## Related Issue
#50 

## Other Information
The updated Readme.md file refers to the state of the bot after #46 is merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,84 +1,191 @@
...
+
+Für die Entwicklung empfehlen wir [IntelliJ](https://www.jetbrains.com/idea/download/) (Community Version reicht aus) als Entwicklungsumgebung. Andere IDE's können auch verwendet werden, folgende Erklärungen beziehen sich jedoch 
+ausschließlich auf IntelliJ.
+
+Beginne damit, eines Projekt zu erstellen. Nutze dazu das Menü `File->New->Poject from Version Control`. 
+Gib dort als URL die URL deines Forks der Repository an. Dadurch wird ein Projekt erstellt, das den Sourcecode sowie die Buildscripts der aktuellen Version des SlimeBallBots enthält.
+In der `Project Structure` des Projekts muss das JDK 17 angeben werden. 
+
+Um den Bot zu starten, musst du zunächst das Token deines Test-Bots in die `.env` Datei als `TEST` token eintragen. 
+Es wird vorausgesetzt, dass du bereits einen Bot Account im [Developer Portal](https://discord.com/developers) erstellt hast, und weißt, wie du dsa Token kopieren kannst.
```
Comment (MatitoDA) [inline README.md:47]: typo bei "das Token".

**Review by MatitoDA — CHANGES_REQUESTED** [review]
just some typos

File: README.md (hunk 1 above)
Comment (MineKing9534) [reply README.md:47]: Was ist da der Fehler? Meinst du "der Token"? Ich glaube beides ist möglich

Comment (MineKing9534) [reply README.md:47]: Ahhh, habs gefunden

**Review by MatitoDA — CHANGES_REQUESTED** [review]
add: if a issue has been assigned to a team member it is already in progress

**Review by MatitoDA — APPROVED** [review]
wait for #46

**Pull request merged**